tee-worker/litentry/core/credentials/src/lib.rs

```diff
@@ -367,6 +408,18 @@ impl Credential {
 
 		self.credential_subject.assertions = assertion;
 	}
+
+	pub fn add_assertion_a8(&mut self, min: u64, max: u64) {
+		let min = format!("{}", min);
+		let max = format!("{}", max);
+
+		let web2_item = AssertionLogic::new_item("$total_txs", Op::GreaterThan, &min);
+		let web3_item = AssertionLogic::new_item("$total_txs", Op::LessEq, &max);
```

Comment:
Changing assertion logic to `$total_txs>=min` and `$total_txs<max` could be better?